000-template.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: post
+title: (fill me in with the episode's title)
+date: (fill me in with today's date, YYYY-MM-DD)
+file: https://audio.rustacean-station.org/file/rustacean-station/(fill me in with the episode mp3 filename)
+duration: (fill in with audio length, HH:MM:SS)
+length: (fill in with audio size in bytes)
+
+# https://github.com/jekyll/jekyll/issues/7744
+layout: episode
+---
+
+<!--
+The episode introduction goes here.
+The first paragraph should ideally be short, and is used in various
+places as a "short description" for the episode. Any subsequent
+paragraphs show up as "expanded description".
+-->
+
+### Contributing to Rustacean Station
+
+<!-- You can probably leave this as-is -->
+
+Get in touch with us if you'd like to be interviewed, propose a topic for an episode, or help out!
+
+ - Twitter: [@rustaceanfm](https://twitter.com/rustaceanfm)
+ - Discord: [Rustacean Station](https://discord.gg/cHc3Gyc)
+ - Github: [@rustacean-station](https://github.com/rustacean-station/)
+ - Email: [hello@rustacean-station.org](mailto:hello@rustacean-station.org)
+
+### Timestamps & referenced resources
+
+<!--
+In this section, leave timestamped notes of the form:
+
+ - [@HH:MM:SS] - Timestamp description
```

Comment:
Does `-` also indicate the creation of an unordered list?

Reply:
Yup, `-` and `*` can both be used to denote unordered lists (see https://commonmark.org/help/). I tend to prefer `-` since `*` _also_ denotes text emphasis.